assets/js/components/InstanceOverview/InstanceOverview.jsx

```diff
@@ -17,14 +19,30 @@ function InstanceOverview({
     features,
     host_id: hostID,
     host,
+    absent_at: absentAt,
   },
 }) {
   const isDatabase = DATABASE_TYPE === instanceType;
-
+  const rowClasses = classNames(
+    { 'bg-gray-100': absentAt },
+    'table-row border-b'
+  );
   return (
-    <div className="table-row border-b">
-      <div className="table-cell p-2">
-        <HealthIcon health={health} />
+    <div className={rowClasses}>
+      <div className="table-cell p-2 px-5">
+        {absentAt ? (
+          <Tooltip content="Instance currently not registered." place="bottom">
+            <span
```

Comment:
Why do we need this double span here? Cannot we simply put the icon component, you can put the `data-testid` there if you want I guess

Reply:
I basically followed how it was done elsewhere, I guess the reasoning is that our Tooltip component doesn't allow `data-testid`. If we add it, wouldn't that override the `data-testid` that EOS already adds by default? (`data-testid="eos-svg-component"`)

Reply:
I think the `Tooltip` usage you based this has other purposes, like using the span to put a text.
There are other examples where we don't have this `span`s.
Here you have an empty span which doesn't do anything.
You can perfectly pass the `data-testid` to `Tooltip` as it receives the all the props as `rest`